src/Symfony/Component/Form/Command/DebugCommand.php

```diff
@@ -87,14 +94,42 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if (null === $class = $input->getArgument('class')) {
             $object = null;
-            $options['types'] = $this->types;
+            $options['core_types'] = $this->getCoreTypes();
+            $options['service_types'] = array_values(array_diff($this->types, $options['core_types']));
             $options['extensions'] = $this->extensions;
             $options['guessers'] = $this->guessers;
+            foreach ($options as &$list) {
```

Comment:
can we get rid of the reference? would be better to me

Reply:
It's required for `sort()` effect.

Reply:
would that work?
```
foreach ($options as $k => $list) {
    sort($options[$k]);
}
```

Reply:
Done.